runtime/sunvmi/sunvmi.c

```diff
@@ -263,10 +263,13 @@ JVM_GetCallerClass_Impl(JNIEnv *env, jint depth)
 	
 	walkState.frameWalkFunction = getCallerClassIterator;
 
-	if (J2SE_18 <= (J2SE_VERSION(vm) & J2SE_VERSION_MASK)) {
-		if (-1 == depth) {
+	if (J2SE_18 <= J2SE_VERSION(vm)) {
```

Comment:
Since OpenJ9 only supports Java 8 or later, do you want to remove this test completely?

Reply:
Sure, removed.
Will use another PR to get rid of other `J2SE_18` checks.